cores/arduino/SERCOM.cpp

```diff
@@ -400,17 +400,20 @@ void SERCOM::disableWIRE()
   }
 }
 
-void SERCOM::initSlaveWIRE( uint8_t ucAddress )
+void SERCOM::initSlaveWIRE( uint8_t ucAddress, bool broadcast )
```

Comment:
Should we call the parameter `listen_on_broadcast` or something similar? @sandeepmistry

Reply:
Maybe `enableGeneralCall`?